### ISSUE TITLE
install issue debian

### ISSUE THREAD

**Issue by ksz00** [paulmcauley/klassy#95]
When I tried to Build and install klassy from source script i got this massage:
make: *** No targets specified and no makefile found.  Stop.
make: *** No rule to make target 'install'.  Stop.

**Comment by paulmcauley** [comment]
Please can you give the exact commands typed?

**Comment by ksz00** [comment]
1st:
sudo apt install build-essential libkf5config-dev libkdecorations2-dev libqt5x11extras5-dev qtdeclarative5-dev extra-cmake-modules libkf5guiaddons-dev libkf5configwidgets-dev libkf5windowsystem-dev libkf5coreaddons-dev gettext cmake libkf5iconthemes-dev libkf5package-dev libkf5style-dev libkf5kcmutils-dev kirigami2-dev

2:
I pressed the <>code button and download zip

3:
I unzipped that, then I opened terminal in the "klassy-master" libary, then I did the Step 2: Then, build and install
Build and install from source script:
./install.sh
and thats what I got:
make: *** No targets specified and no makefile found. Stop.
make: *** No rule to make target 'install'. Stop.
